UPGRADE.md

```diff
@@ -15,18 +55,18 @@ To upgrade from an already-existing Ilios 2.4.8 installation, perform the follow
 1. Backup all of your learning materials or move them to a location where they will be accessible by the new Ilios 3 installation. *WARNING:* If your learning materials are stored in the default location, be careful to not accidentally delete them when you change your codebase.
 2. Backup your current database completely, and do not forget to add the '-R' or'--routines' flags to ensure that your stored procedures and triggers are included in the back up. The command would probably look something like this:
 
-```bash
+ ```bash
```

Comment:
There should be now space before the first tick-mark (`), otherwise markdown doesn't render the syntax...